lib/main.rb

```diff
@@ -0,0 +1,87 @@
+# main.rb
+require_relative 'planet'
+require_relative 'solar_system'
+
+#This method should create two different instances of Planet and print out some of their attributes.
+
+def main
```

Comment:
This method begs to be broken up into smaller methods.

Reply:
Okay, thanks I thought everything had to stay into main, that is the only reason why I didn't. Now I know for future.